lib/swap.js

```diff
@@ -154,15 +154,18 @@ swap.submit = function(e, selectors) {
       const form = document.createElement('form');
       form.method = target.getAttribute('formmethod');
       form.action = formaction;
-
-      const body = JSON.parse(decodeURIComponent(target.getAttribute('formbody')));
-      Object.keys(body).forEach(key => {
-        const input = document.createElement('input');
-        input.type = 'hidden';
-        input.value = body[key];
-        input.name = key;
-        form.append(input);
-      });
+      const formbody = target.getAttribute('formbody');
```

Comment:
It was difficult to know where to position this line. If you want it somewhere else, just let me know.

Reply:
I'd bump it under line 154 since that's a const too